airbyte-integrations/connectors/destination-duckdb/destination_duckdb/destination.py

```diff
@@ -99,17 +130,24 @@ def write(
                 logger.info(f"Dropping tables for overwrite: {table_name}")
                 query = f"DROP TABLE IF EXISTS {schema_name}.{table_name}"
                 con.execute(query)
+
+            # Get the SQL column definitions
+            sql_columns = self._get_sql_column_definitions(configured_stream)
+            column_definition_str = ",\n                ".join(
+                f"{self._quote_identifier(column_name)} {sql_type}"
+                for column_name, sql_type in sql_columns.items()
+            )
+
             # create the table if needed
             query = f"""
             CREATE TABLE IF NOT EXISTS {schema_name}.{table_name} (
-                _airbyte_ab_id TEXT PRIMARY KEY,
-                _airbyte_emitted_at DATETIME,
-                _airbyte_data JSON
+                {column_definition_str}
```

Comment:
Nice. 👍 For cases where a table already exists but doesn't have all of the latest columns, we will need to make sure we add is detecting missing columns and adding those to the table if needed. In PyAirbyte, this is handled by [`_add_missing_columns_to_table()`](https://github.com/airbytehq/PyAirbyte/blob/4777d2407e5329a968e7a7fd7feac80088e4f6af/airbyte/shared/sql_processor.py#L883-L916).